extensions/src/AWSSDK.Extensions.CloudFront.Signers/AWSSDK.Extensions.CloudFront.Signers.NetFramework.csproj

```diff
@@ -0,0 +1,45 @@
+<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFramework>net472</TargetFramework>
+		<AssemblyName>AWSSDK.Extensions.CloudFront.Signers</AssemblyName>
+		<PackageId>AWSSDK.Extensions.CloudFront.Signers</PackageId>
+		<Version>4.0.0.0-preview</Version>
+		<Title>AWSSDK.Extensions.Cloudfront.Signers</Title>
+		<Description>
```

Comment:
The indentation was done using `ctrl K + f`, but good point it does looks different, I will adjust accordingly and also remove the description since it will come from the `nuspec` file.